config/clusters/2i2c/ucmerced.values.yaml

```diff
@@ -27,8 +27,8 @@ jupyterhub:
           url: http://www.ucmerced.edu/
   singleuser:
     image:
-      name: quay.io/2i2c/2i2c-hubs-image
-      tag: "14107b8a85fb"
+      name: quay.io/2i2c/culler-failure
```

Comment:
This should actually be setting the image under `hub.image` rather than `singleuser.image`

Reply:
I opened https://github.com/2i2c-org/infrastructure/issues/3150 about having a separate staging hub for ucmerced perhaps, given they're still on the old user image and there may be other things we need to test.

Reply:
Woof, is it too late in the game to change that image name? I was definitely lead to believe I was changing the right thing since it was the "hubs-image"